examples/data-objects/multiview/interface/package.json

```diff
@@ -11,13 +11,13 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"type": "module",
 	"main": "lib/index.js",
-	"module": "lib/index.js",
 	"types": "lib/index.d.ts",
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
-		"build:esnext": "tsc",
+		"build:esnext": "tsc --project ./tsconfig.json",
```

Comment:
@jason-ha - I'm not sure when the '--project ./tsconfig.json' is necessary, but added it for consistency.

Reply:
`./tsconfig.json` is the default project. I had added it in a number of multi-build cases to make it easier to see which was in use.